### ISSUE TITLE
ChildBrowser: Access to page title

### ISSUE THREAD

**Issue by frodeaa** [purplecabbage/phonegap-plugins#48]
It would be useful to get access to page title when using Google OAuth 2.0 for Installed Applications, i.e PhoneGap applications.

Maybe extend onLocationChange to include an array with additional parameters...

Google Using OAuth 2.0 for Installed Applications:
 http://code.google.com/intl/no-NO/apis/accounts/docs/OAuth2InstalledApp.html

**Comment by gduquesnay** [comment]
Yep.  I need this too. Did you ever find a way to work around it or are you making users copy and paste the token?
